networkx/readwrite/gml.py

```diff
@@ -709,9 +745,9 @@ def stringize(key, value, ignored_keys, indent, in_list=False):
                 yield indent + ']'
             elif isinstance(value, (list, tuple)) and key != 'label' \
                     and value and not in_list:
-                next_indent = indent + '  '
+                yield indent + key + ' ' + '"{}"'.format(LIST_START_VALUE)
```

Comment:
think we should make this conditional on the list length being only 1, since it isn't necessary for longer lists